### PR TITLE
fix positioning issue with IE

### PR DESCRIPTION
**Pull request by ajwild**
This fix is being used in a few other forks of ng-context-menu/angular-context-menu. See https://github.com/ColourboxDevelopment/angular-context-menu/commit/a122c17c2116bae84c59d959c5335c0666cc1aaa as one example.

It's not just for IE8, it also helps with events generated with `document.createEvent('MouseEvent').initMouseEvent()` in later versions of IE and Edge.

This fix is fairly common and the way the if statement is setup means it will have no impact on most browsers. See [these](http://stackoverflow.com/questions/11245119/how-to-get-mouse-pointer-position-using-javascript-for-internet-explorer) [three](http://stackoverflow.com/questions/30347005/mouse-position-in-ie) [links](http://stackoverflow.com/questions/982430/event-pagex-use-jquery-event-in-a-function-not-bound-through-jquery) for StackOverflow questions/answers relating to the fix. In this particular scenario it is a little bit strange to add docLeft and then remove it again, but it seemed like the safest way to fix the bug with minimal risk.

Let me know if you want me to bump the version before merging.

### CODE REVIEW

File: src/ng-context-menu.js [hunk 1]
```diff
@@ -40,12 +40,22 @@
                   (doc.clientTop || 0),
                 elementWidth = menuElement[0].scrollWidth,
                 elementHeight = menuElement[0].scrollHeight;
+              var pageX;
+              var pageY;
+              if (event.pageX || event.pageY) {
```
Comment (pads) [inline src/ng-context-menu.js:46]: Can you put in comment for future readers just to explain why this is here please?

**Comment by pads** [comment]
Thanks a lot for this contribution! Bump the version and I'll be happy to merge.

**Comment by ajwild** [comment]
@pads I've updated the commit. It looks like adding `(function(angular) {` in another commit has changed the indenting so most lines in dist/ng-context-menu.js have been updated. I'm not sure what has happened since the last release, so if you would prefer version 1.1.0 I will update it again. Also, if you prefer the "fix" commit and a separate "bump" commit I can do that too.

**Comment by pads** [comment]
@ajwild this looks fine. I don't think the dist task was run properly/at all for the last release. Looking at the diff it's formatting only plus your changes so the version number is fine.

**Pull request merged**